Marlin/Conditionals.h

```diff
@@ -46,6 +46,21 @@
 
   #define CONFIGURATION_LCD
 
+  #if ENABLED(CartesioUI)
+    #define DOGLCD
+    #define SDSUPPORT
```

Comment:
Don't define `SDSUPPORT` here. We currently require users to enable this option manually. In future we may go back to specifying which controllers have SD support, with a `DISABLE_SDSUPPORT` option to disable it. But for now, it is a single option so that users can disable it in `Configuration.h` just by uncommenting it.